src/client/driver/command-executors/client-functions/client-function-executor.js

```diff
@@ -43,12 +46,9 @@ export default class ClientFunctionExecutor {
             });
     }
 
-    // Overridable methods
-    _createReplicator () {
-        return createReplicator([
-            new ClientFunctionNodeTransform(this.command.instantiationCallsiteName),
-            new FunctionTransform()
-        ]);
+    //Overridable methods
+    _addReplicatorTransform () {
```

Comment:
I'm not sure I understood your comment. If I add `ClientFunctionTransform` in constructor of the base class `ClientFunctionExecutor`, I can't add `SelectorTransform` in `SelectorExecutor` constructor, because we can't add two transforms with the same type `Node` to the replicator.